From: Sebastian Ramacher <sramacher@debian.org>
Date: Mon, 17 Jul 2023 23:24:06 +0200
Subject: Bump required C++ version to 14
Bug: https://github.com/acoustid/chromaprint/issues/132

---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index eae99d4..501f182 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -24,7 +24,7 @@ find_package(Threads)
 
 option(BUILD_SHARED_LIBS "Build shared libraries" ON)
 
-set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_STANDARD 14)
 
 if(BUILD_SHARED_LIBS)
 	set(CMAKE_C_VISIBILITY_PRESET hidden)
